src/Translatable.php

```diff
@@ -188,9 +188,9 @@ protected function createTranslatedField(Field $originalField, string $locale):
 
         $translatedField
             ->resolveUsing(function ($value, Model $model) use ($translatedField, $locale, $originalAttribute) {
-                $translatedField->attribute = 'translations_'.$originalAttribute.'_'.$locale;
-                $translatedField->panel = $this->panel;
-                $translatedField->assignedPanel = $this->assignedPanel;
+                $translatedField->attribute = 'translations_' . $originalAttribute . '_' . $locale;
+                $translatedField->panel ??= $this->panel;
```

Comment:
Sorry for the style changes I accidentally commited. I reverted them and also now use the `??` instead of `??=`. I know this is a little less beautiful, but I don’t think it legitimates requiring php@7.4. What do you think?